src/Validator/Constraints/EventMemberValidator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Validator\Constraints;
+
+use App\Entity\Member;
+use App\Entity\PartyEvent;
+use App\Repository\MemberRepository;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class EventMemberValidator extends ConstraintValidator
+{
+    public function __construct(private readonly MemberRepository $memberRepository)
+    {
+    }
+
+    public function validate($value, Constraint $constraint): void
```

Comment:
### 🤓 Nitpick: `$value` is a bad name.

Maybe some `$member` will be better to understand